docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = python -msphinx
```

Comment:
Does there need to be a space after `-m`?

Reply:
Hmmm that is what I was thinking. I noticed the `Makefile` for `e3sm-diags` is also missing a space.

The default when you run `sphinx-build` looks like `SPHINXBUILD = sphinx-build`. 
 I'm not entirely sure how it is used.

According to the docs:

>https://www.sphinx-doc.org/en/1.0/invocation.html#makefile-options 
SPHINXBUILD
The command to use instead of sphinx-build.